src/main/java/com/cburch/logisim/analyze/gui/Analyzer.java

```diff
@@ -135,37 +135,6 @@ private void update() {
     }
   }
 
-  public static void main(String[] args) throws Exception {
```

Comment:
Why do you remove this member function? Is it outdated? It seems allowing to test the `Analyzer` class.

Reply:
I guess I most likely managed to remove one line too much and that line was folded method. It is interesting to know how come I did not notice that while reviewing my patch for PR, but I am afraid I got no answer ATM. Method restored.

Reply:
Method restored. Thanks for paying attention and actually reviewing the PRs.

Reply:
Thanks! You are welcome!